examples/tutorial_client_eventfilter.c

```diff
@@ -0,0 +1,267 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+/**
+ * In this C file we will create a ClientFilter example.
+ * Client configured on the server
+ * */
+
+#include <open62541/client.h>
+#include <open62541/client_config_default.h>
+#include <open62541/client_subscriptions.h>
+#include <open62541/plugin/log_stdout.h>
+#include <open62541/util.h>
+#include <signal.h>
+#ifdef _MSC_VER
+#pragma warning(disable:4996) // warning C4996: 'UA_Client_Subscriptions_addMonitoredEvent': was declared deprecated
+#endif
+
+#ifdef __clang__
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
+
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
+
+static UA_Boolean running = true;
+const size_t nSelectClauses = 2;
+const size_t nWhereClauses = 1;
+
+static void stopHandler(int sig) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "received ctrl-c");
+    running = false;
+}
+
+#ifdef UA_ENABLE_SUBSCRIPTIONS
+
+static void
+handler_events(UA_Client *client, UA_UInt32 subId, void *subContext,
+               UA_UInt32 monId, void *monContext,
+               size_t nEventFields, UA_Variant *eventFields) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "Notification");
+
+    /* The context should point to the monId on the stack */
+    UA_assert(*(UA_UInt32*)monContext == monId);
+
+    for(size_t i = 0; i < nEventFields; ++i) {
+        if(UA_Variant_hasScalarType(&eventFields[i], &UA_TYPES[UA_TYPES_UINT16])) {
+            UA_UInt16 severity = *(UA_UInt16 *)eventFields[i].data;
+            UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "Severity: %u", severity);
+        } else if (UA_Variant_hasScalarType(&eventFields[i], &UA_TYPES[UA_TYPES_LOCALIZEDTEXT])) {
+            UA_LocalizedText *lt = (UA_LocalizedText *)eventFields[i].data;
+            UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                        "Message: '%.*s'", (int)lt->text.length, lt->text.data);
+        }
+        else {
+#ifdef UA_ENABLE_TYPEDESCRIPTION
+            UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                        "Don't know how to handle type: '%s'", eventFields[i].type->typeName);
+#else
+            UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                        "Don't know how to handle type, enable UA_ENABLE_TYPEDESCRIPTION "
+                        "for typename");
+#endif
+        }
+    }
+}
+
+/**
+ * Setting up WhereClauses
+ * ^^^^^^^^^^^^^^^^^^^
+ * The WhereClause defines the filtering criteria of an EventFilter.
+ * It is represented in a ContentFilter Structure.
+ * The ContentFilter structure defines a collection of elements,
```

Comment:
please define "elements" here more prcise. Events? Fields?..

Reply:
"Element" refers to the ContentFilterElement and is described more precisely in the next sentence. Nevertheless, we changed "elements" to "ContentFilterElements" now to avoid any confusion.